src/CommunityToolkit.Maui/Extensions/PropertyChangedEventArgsExtensions.shared.cs

```diff
@@ -0,0 +1,46 @@
+//inspired from https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/CollectionView/PropertyChangedEventArgsExtensions.cs
+
+using System.Runtime.CompilerServices;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+static class PropertyChangedEventArgsExtensions
+{
+	[MethodImpl(MethodImplOptions.AggressiveInlining)]
+	public static bool Is(this string propertyName, BindableProperty property) =>
+		propertyName == property.PropertyName;
+
+	[MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Can we use param and compare all items at once 
properties.All(p=> p.PropertyName==propertyName)

Reply:
params allocate an array, for small numbers of properties isn't efficient. I believe that was the reason that XF team didn't use it

Reply:
In terms of maintainability and ease to use I would prefer params. the performance improvement is so small.
Up to you. Approved

Reply:
Well I just copied and pasted it, we shouldn't touch this code anymore (I hope), if so we can add the overload with params